### ISSUE TITLE
Loss function mistake

### ISSUE THREAD

**Issue by Jiawei0o0** [ycwu1997/MC-Net#4]
Thanks for your great work. 
The code may have a little mistake in the 'train_mcnet_3d.py' file, line 163. The  'loss_seg' （ line 146） is not count in the backward function.  Or it just not take part in it? Am I missing something here?

**Comment by ycwu1997** [comment]
Thanks for your interests. Actually, in this paper, We only use the dice loss, named as Loss_seg_dice, for the model training. Here, the CE loss is just computed but not introduced for the training. You can check the paper for the details. Thanks.
